docs/apidoc/watertap.core.rst

```diff
@@ -0,0 +1,17 @@
+watertap.core package
+=====================
+
+Subpackages
+-----------
+
+.. toctree::
+   :maxdepth: 4
+
+   watertap.core.util
+
+Module contents
+---------------
+
+.. automodule:: watertap.core
+   :members:
+   :show-inheritance:
```

Comment:
Why are all these new documents being put into this PR? Was this necessary to get the checks to pass?